### PR TITLE
add fcgi2-2.4.2 build files

### PR DESCRIPTION
**Pull request by pbdigital-dot-org**
add library/fcgi2 & application/fcgiwrap for nginx/nagios

I have Nagios running and packaged (with nagios-plugins,nrdp & nsca), which I would like to push up to omnios-extra.
Nagios is running under nginx and requires fcgiwrap, so I have made PRs for the fcgi parts today. Any pointers I get on these PRs 
I can apply to the Nagios packages so as not to consume too much of your time.

for sure, one thing I was not certain on was UID & PID for user fcgiwrap. 

Philip

### CODE REVIEW

**Comment by hadfl** [comment]
Hi @pbdigital-dot-org 

Thank you for yet another contribution! Again, your PR looks all nice and clean.

I've collected some suggested changes in https://github.com/hadfl/omnios-extra/commit/f028a8f7df5569c7b2652da08f0b633fe341f888 to:

- drop the patch to configure in favour of setting `CONFIG_SHELL`
- we usually ship libraries as multi-arch packages

**Comment by pbdigital-dot-org** [comment]
great! I will sort this out tomorrow morning.

File: doc/baseline [hunk 1]
```diff
@@ -1,3 +1,4 @@
+extra.omnios ooce/application/fcgi2
```
Comment (hadfl) [inline doc/baseline:1]: oh, and just realised that I overlooked this. It should be `ooce/library/fcgi2`

File: doc/packages.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 | Package | Version | Link | Maintainer |
 | :------ | :------ | :--- | :--------- |
+| ooce/application/fcgi2	| 2.4.2		| https://github.com/FastCGI-Archives/fcgi2/releases | [omniosorg](https://github.com/omniosorg)
```
Comment (hadfl) [inline doc/packages.md:3]: same as above.

**Comment by pbdigital-dot-org** [comment]
ok, I have applied those changes and pushed them up, onto fcgiwrap, thanks!

**Comment by hadfl** [comment]
Great, thanks a lot. Just one last thing: could you please move the entries in `doc/packages.md` and `doc/baseline` so we keep everything in alphabetic order?

**Pull request merged**